DeusEx/Classes/DXRPasswords.uc

```diff
@@ -61,6 +83,57 @@ function RandoPasswords(int mode)
     }
 }
 
+function RandoInfoDevs(int percent)
+{
+    local InformationDevices id;
+    local Inventory inv;
+    local int i, num, slot;
+    local int hasPass[64];
+    local DeusExTextParser parser;
+
+    if(percent == 0) return;
+
+    foreach AllActors(class'InformationDevices', id)
+    {
+        if( rng(100) > percent ) continue;
+        
+        for(i=0; i<ArrayCount(hasPass); i++)
+            hasPass[i]=0;
+        
+        if ( id.textTag != '' ) {
```

Comment:
if there's no textTag then is there any reason to swap it with something? do any of these exist in the game?